tests/acceptance/test_author_creation.py

```diff
@@ -0,0 +1,174 @@
+    'institution-name': 'CERN',
+    'institution-start_year': '2000',
+    'institution-end_year': '2001',
+    'institution-rank': 'STAFF',
+    'experiments-name': 'ATLAS',
+    'experiments-start_year': '2002',
+    'experiments-end_year': '2005',
+    'advisors-name': 'Bob White',
+    'advisors-degree_type': 'PhD',
+    'comments': 'Some comments about the author',
+    'subject-0': 'ACC-PHYS',
+    'subject-1': 'ASTRO-PH'
+}
+
+
+def test_institutions_typehead(login):
+    create_author.go_to()
+    assert create_author.write_institution('cer', 'CERN').has_error()
+
+
+def test_experiments_typehead(login):
+    create_author.go_to()
+    assert create_author.write_experiment('atl', 'ATLAS').has_error()
+
+
+def test_advisors_typehead(login):
+    create_author.go_to()
+    assert create_author.write_advisor('alexe', 'Vorobyev, Alexey').has_error()
+
+
+def test_mail_format(login):
+    create_author.go_to()
+    assert create_author.write_mail('wrong.mail').has_error()
+    assert not create_author.write_mail('me@me.com').has_error()
+
+
+def test_ORCID_format(login):
+    create_author.go_to()
+    assert create_author.write_orcid('wrong.ORCID').has_error()
+    assert not create_author.write_orcid('1111-1111-1111-1111').has_error()
+
+
+def test_institutions_years(login):
+    create_author.go_to()
+
+    input_id = 'institution_history-0-start_year'
+    error_mess_id = 'state-institution_history-0-start_year'
+    assert create_author.write_year(input_id, error_mess_id, 'wrongyear').has_error()
+    assert not create_author.write_year(input_id, error_mess_id, '2016').has_error()
+
+    input_id = 'institution_history-0-end_year'
+    error_mess_id = 'state-institution_history-0-end_year'
+    assert create_author.write_year(input_id, error_mess_id, 'wrongyear').has_error()
+    assert not create_author.write_year(input_id, error_mess_id, '2016').has_error()
+
+
+def test_experiments_years(login):
+    create_author.go_to()
+
+    input_id = 'experiments-0-start_year'
+    error_mess_id = 'state-experiments-0-start_year'
+    assert create_author.write_year(input_id, error_mess_id, 'wrongyear').has_error()
+    assert not create_author.write_year(input_id, error_mess_id, '2016').has_error()
+
+    input_id = 'experiments-0-end_year'
+    error_mess_id = 'state-experiments-0-end_year'
+    assert create_author.write_year(input_id, error_mess_id, 'wrongyear').has_error()
+    assert not create_author.write_year(input_id, error_mess_id, '2016').has_error()
+
+
+def test_mandatory_fields(login):
+    expected_data = {
+        'given-name': 'This field is required.',
+        'display-name': 'This field is required.',
+        'reserach-field': 'This field is required.'
+    }
+
+    create_author.go_to()
+    assert create_author.submit_empty_form(expected_data).has_error()
+
+
+def test_submit_author(login):
+    create_author.go_to()
+    assert create_author.submit_author(input_author_data).has_error()
```

Comment:
So in all of these `has_error()` is `True` but still they continue to reach holding pen?